src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  let totalCost = cost * days;
```

Comment:
Use 'const' for 'totalCost' instead of 'let' since it's not being re-assigned but only modified within the 'if' blocks.